Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -1170,27 +1170,37 @@ final class PackageToolTests: CommandsTestCase {
     func testCommandPlugin() throws {
```

Comment:
is it possible to add a test for the binary dependency case?

Reply:
Definitely, I actually have that in a later PR, but can bring that here.